pkg/antctl/raw/traceflow/command.go

```diff
@@ -77,9 +73,9 @@ func init() {
 		Example: `  Start a Traceflow from busybox0 to busybox1, both pods are in namespace default
   $antctl traceflow -S busybox0 -D busybox1
   Start a Traceflow from busybox0 to destination IP, source is in namespace default
-  $antctl traceflow -S busybox0 -D 123.123.123.123
+  $antctl traceflow -S busybox0 -D 123.123.123.123 -T ip
```

Comment:
I was thinking we could follow `kubectl exec` which has same situation.
```
kubectl exec (POD | TYPE/NAME) [-c CONTAINER] [flags] -- COMMAND [args...] [options]

# Get output from running 'date' command in ruby-container from pod mypod
kubectl exec mypod -c ruby-container -- date

# Get output from running 'date' command from the first pod of the service myservice, using the first container by
default
kubectl exec svc/myservice -- date
```
But it looks like it would be more complex as we already use namespace as the prefix.

Adding "-T" to indicate the type of destination looks a little unnecessary to me, let's see others' opinions. Or we could defer this change first and discuss it with the same case of trace-packet command together.

Reply:
Yes, our `antctl traceflow` have source and destination with 2 namespaces, which is different from `kubectl exec`. But it should be handled that pod and service could have same name. So I think change in this PR should be merged for now.